test/ignores.json

```diff
@@ -1,46 +0,0 @@
-{
```

Comment:
Could you make sure and confirm that we don't add extra work to -native with this change? Since this file is also used in native there's a risk to moving away from `.json` to `.js` format, and it might not be worth diverging. The benefit we'd get from using `.js` for multi-line comment is minimal compared to the eventual effort to make to move away from parsing json to parsing js.

Refer to some of their parsing code here for their test suite: https://github.com/mapbox/mapbox-gl-native-internal/blob/f08b9dc864cb34c34aae5d0e702ab3ca770d42bb/render-test/manifest_parser.cpp#L65, but it's unclear whether they read this file or other ignore files, as they have a lot of them in other contexts.

Reply:
I don't think they read this file considering it mentions features that need to be ported from Native to GL JS, and the Native equivalent is [here](https://github.com/mapbox/mapbox-gl-native-internal/blob/f08b9dc864cb34c34aae5d0e702ab3ca770d42bb/metrics/ignores/platform-all.json). But I'm double-checking with the Native team to confirm this and ensure that there won't be other disadvantages to diverging ignore formats.

Reply:
Sounds good!